plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -85,20 +308,55 @@
         template: "#remote-config-parameters",
         mixins: [countlyVue.mixins.hasDrawers("parameters")],
         components: {
-            drawer: ParametersDrawer
+            drawer: ParametersDrawer,
+            "json-editor": JsonEditor
+
         },
         computed: {
             tableRows: function() {
                 return this.$store.getters["countlyRemoteConfig/parameters/all"];
             }
         },
         methods: {
+            getNumberOfConditionsText: function(conditions) {
+                if (conditions.length === 0) {
+                    return "1 condition";
+                }
+                return conditions.length + 1 + "conditions";
```

Comment:
why are you adding 1 to the conditions.length?

Reply:
This is for default value, which is a mandatory field

Reply:
default value is a mandatory field i agree.
but why is it counted in the conditions?
default value of a parameter has nothing to do with the conditions or its values.

Reply:
This was according to design. I think we can discuss with Duygu and remove it:) 

https://www.figma.com/file/PUjfGUiaVQpSBeS5EWMKYu/countly-components?node-id=11374%3A77951